docs/website/docs/dlt-ecosystem/llm-tooling/mcp-server.md

```diff
@@ -0,0 +1,202 @@
+
+This page gives an overview of what we're building and includes detailed instructions to install the MCP in your favorite IDE.
+
+:::caution
+🚧 This feature is under development, and the interface may change in future releases. Interested in becoming an early tester? [Join dlt+ early access](https://info.dlthub.com/waiting-list).
+:::
+
+## Model Context Protocol
+The [Model Context Protocol](https://modelcontextprotocol.io/introduction) (MCP) is a standard initiated by Anthropic to connect large language models (LLMs) to external data and systems.
+
+In the context of the MCP, the **client** is built-in the user-facing application. The most common clients are LLM-enabled IDEs or extensions such as Continue, Cursor, Claude Desktop, Cline, etc. The  **server** is a process that handles requests to interact with external data and systems.
+
+### Core constructs
+
+- **Resources** are data objects that can be retrieved by the client and added to the context (i.e., prompt) of the LLM request. Resources will be manually selected by the user, or certain clients will automatically retrieved them.
+
+- **Tools** provided a way to execute code and provide information to the LLM. Tools are called by the LLM; they can't be selected by the user or the client. 
+
+- **Prompts** are strings, or templated strings, that can be injected in the conversation. Prompts are selected by the user. They provide shortcuts for frequent commands, or allow to ask the LLMs to use specific tools.
+
+
+:::note
+The MCP is progressively being adopted and not all clients support all the features used by the dlt MCP server. See [this page](https://modelcontextprotocol.io/clients) for an overview of client capabilities
+:::
+
+
+## Features
+
+The `dlt` and `dlt+` MCP servers aim to be a toolbox to help developers build, maintain, and operate `dlt` pipelines. There are two primary avenues:
+
+- **Code generation**: LLMs are good at writing Python code, but they don't know everything about `dlt`. The MCP provides resources and tools to provide up-to-date information to the LLm about the dlt library and the specifics of your project.
+
+- **Assistance**: The MCP resources, tools, and prompts can also be used to learn more about `dlt`. The built-in knowledgebase combined with your project's context brings Q&A support inside your IDE.
+
+The next sections are a non-exhaustive documentation of existing and upcoming features.
+
+### Tools
+
+- Use your local pipeline directory (default: `~/.dlt/pipelines`) to retrieve pipeline metadata. This includes: available pipleines, available tables, table schemas. It also include operational information such as last load date, schema changes, schema diffs, load packages data.
```

Comment:
do these tools have names? It would make it more memorable

I can imagine 
- PipelineMetadata
- PipelineConfigData
- DestinationData